go/libraries/doltcore/sqle/dprocedures/dolt_checkout.go

```diff
@@ -43,13 +43,13 @@ func doltCheckout(ctx *sql.Context, args ...string) (sql.RowIter, error) {
 }
 
 func doDoltCheckout(ctx *sql.Context, args []string) (int, error) {
-	dbName := ctx.GetCurrentDatabase()
-
-	if len(dbName) == 0 {
-		return 1, fmt.Errorf("Empty database name.")
+	revDbName := ctx.GetCurrentDatabase()
+	if len(revDbName) == 0 {
+		return 1, fmt.Errorf("Empty database branchName.")
```

Comment:
`revDbName` seems a little misleading here, since the current database doesn't have to be a revision database. Similar comments for the error message string mentioning `database branchName`.